ui/src/jobs/JobShow.tsx

```diff
@@ -32,6 +36,53 @@ const SuccessField = (props: any) => {
     return (props.record["finished_at"] === null ? <Tooltip title="Running"><JobIcon /></Tooltip> : <BooleanField {...props} />);
 };
 
+const FullButton = ({record}: any) => {
+    const dispatch = useDispatch();
+    const notify = useNotify();
+    const [loading, setLoading] = useState(false);
+    const handleClick = () => {
+        setLoading(true);
+        dispatch(fetchStart()); // start the global loading indicator 
+        fetch(`${apiUrl}/jobs/${record.job_name}/executions/${record.id}`)
+            .then((response) => {
+                if (response.ok) {
+                    notify('Success loading full output');
+                    return response.json()
+                }
+                throw response
+            })
+            .then((data) => {
+                record.output_truncated = false
+                record.output = data.output
+            })
+            .catch((e) => {
+                notify('Error on loading full output', 'warning')
+            })
+            .finally(() => {
+                setLoading(false);
+                dispatch(fetchEnd()); // stop the global loading indicator
+            });
+    };
+    return (
+        <Button 
+            label="Load full output"
+            onClick={handleClick}
+            disabled={loading}
+        >
+            <FullIcon/>
+        </Button>
+    );
+};
+
+const SpecialOutputPanel = ({ id, record, resource }: any) => {
+    return (
+        <div className="execution-output">
+            {record.output_truncated ? <div><FullButton record={record} /></div> : ""}
+            {record.output || "Empty output"}
```

Comment:
This will display "Empty output" as well when outputsizelimit=0 but the actual output size is > 0.
A minor thing, but still.

Reply:
hmm not sure what to show then. if it's truncated to 0 or it has no output, there's nothing to show there (and I assume the origin `empty output` message is to avoid leaving a empty box which looks "unfinished").

truncated or actually 0 would already be reflected by the `FullButton` presence. Maybe just changing the text to `Nothing to show` makes more sense, rather than adding some condition and alternate text?